readme.md

```diff
@@ -819,6 +820,92 @@ Type: `string`
 
 The IP address used to send the request from.
 
+### HTTPS
+
+##### ca
+
+Type: `string` | `string[]` | `Buffer` | `Buffer[]`
```

Comment:
```suggestion
Type: `string | string[] | Buffer | Buffer[]`
```

Does this mean that you cannot have an array of string **and** buffer?

Reply:
> Does this mean that you cannot have an array of string and buffer?

Sorry, I missed this line.

Node.JS documentation.
> or an Array of strings and/or Buffers

It's ambiguous, but, after some testing, I can say that it accepts both mixed in the same array.
The type should be `string | Buffer | (string | Buffer)[]`